tools/packaging/common/istio-start.sh

```diff
@@ -55,7 +55,7 @@ if [ -n "${ISTIO_CP_AUTH:-}" ]; then
 fi
 
 if [ -z "${ISTIO_SVC_IP:-}" ]; then
-  ISTIO_SVC_IP=$(hostname --ip-address)
+  ISTIO_SVC_IP=$(hostname --all-ip-addresses | cut -d ' ' -f 1)
```

Comment:
Some comment or explanation ? This seems a bit risky, is this option working on ubuntu/etc ?

Reply:
`--ip-address` does not work well on a centos vm with more than one ip address. `--all-ip-addresses` has more predictable output on all systems I've tested (debian, ubuntu, centos).